python_modules/libraries/dagster-airbyte/dagster_airbyte_tests/integration/test_managed_elements.py

```diff
@@ -269,11 +301,19 @@ def test_change_source_and_destination(empty_airbyte_instance, airbyte_source_fi
     check_result = check(TEST_ROOT_DIR, "example_airbyte_stack:reconciler_different_dest")
     assert check_result == expected_diff
 
+    # Apply new destination config, ensure that we get the proper diff and that the call count
+    # stays the same since we are just reconfiguring the same destination type
```

Comment:
should you also check the number of calls to `/destinations/update` here to make sure it goes up? or is just checking the apply result sufficient to tell you that everything is in the proper state

Reply:
It's a good sanity check, I'll add it